pymongo/client_session.py

```diff
@@ -425,7 +448,7 @@ def callback(session):
             :mod:`~pymongo.read_preferences` for options.
 
         :Returns:
-          The return value of the callback.
+          The return value of the ``callback``.
 
```

Comment:
Should we add a `:Raises:` section to this as well? Might be useful to have a quick-reference of all the failure modes.

Reply:
We could be it would just say `Exception` (from the callback) and `PyMongoError` so I'm not sure how useful that would be